src/Periodic/ExtraSmallPeriodicHTMLFormatter.php

```diff
@@ -44,14 +43,14 @@ public function format(Offer $offer): string
     private function formatStarted(DateTimeInterface $endDate): string
     {
         return
-            '<span class="to meta">' . ucfirst($this->trans->getTranslations()->t('till')) . '</span> ' .
+            '<span class="to meta">' . ucfirst($this->translator->getTranslations()->t('till')) . '</span> ' .
             $this->formatDate($endDate);
     }
 
     private function formatNotStarted(DateTimeInterface $startDate): string
     {
         return
-            '<span class="from meta">' . ucfirst($this->trans->getTranslations()->t('from_period')) . '</span> ' .
+            '<span class="from meta">' . ucfirst($this->translator->getTranslations()->t('from_period')) . '</span> ' .
```

Comment:
It seems like we're (almost?) always doing `$this->translator->getTranslations()->t()`. A possible improvement could be to add a `t()` on our own `Translator`, so the underlying translation class becomes an implementation detail? Then these methods calls also become less repetitive and easier to read. Also, http://wiki.c2.com/?LawOfDemeter

Can be done in a follow-up PR of course since this is not necessarily in scope of injecting the `Translator` class instead of creating it in every constructor.

Reply:
Yes, good plan. Will clean this up

Reply:
Cleaned it up, but removed another method in the process. Now that we have 2 breaking changes, I would probably opt for tagging a major version.